DigitalLearningSolutions.Web/Helpers/PromptHelper.cs

```diff
@@ -206,6 +207,17 @@ public static string GetDelegateRegistrationPromptAnswerName(int customPromptNum
             };
         }
 
+        public static string GetCourseAdminFieldAnswerName(int customPromptNumber)

```

Comment:
3 changes which I think would be good.
1. Can we move this onto CourseDelegate as a static method? It's very specific to that class.
2. Can we rename it to make it clear that it's getting the name of the property? `GetPropertyNameForAdminFieldAnswer`
3. customPromptNumber -> **course**PromptNumber

Reply:
All done, I also did the same for the equivalent method for registration prompts which I copied that method from.